app/views/organizations/adopter_fosterer/adopted_pets/index.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= render DashboardPageComponent.new do |c| %>
-  <% c.with_header_title { "Adopted Pets" } %>
+  <% c.with_header_title { t(".header_title") } %>
```

Comment:
Interesting…does Rails find the correctly scoped header title for this? I have not done it this way before.

Reply:
Yes, Rails supports this, and it's called [Lazy Lookup](https://guides.rubyonrails.org/i18n.html#lazy-lookup). Rails automatically scopes the translation key based on the current controller and action, which is why I only need to use t(".header_title") here.

Reply:
Very handy! Thanks for the explanation